clients/go/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"goclients/cacclient"
+	"goclients/expclient"
+)
+
+func main() {
+	// Example usage
+	tenant := "dev"
+	pollingFrequency := 1
+	cachostName := "http://localhost:8080"
+
+	client := cacclient.NewCacClient(tenant, pollingFrequency, cachostName)
+	client.CreateNewCacClient()
```

Comment:
Why do we create a client twice? Can we do these operations in one call?

Reply:
done